vm_supervisor/migrations/script.py.mako

```diff
@@ -0,0 +1,24 @@
+"""${message}
```

Comment:
The `mako` template engine should appear in the frozen `packaging/requirements*` files.

Reply:
I'll update the requirements.